### DISCUSSION TITLE
Unable to update to Shiny 2.2 Xam Forms Android

### DISCUSSION THREAD

**Discussion by jmichas** [shinyorg/shiny#674] [Q&A] [answered]
How am I supposed to resolve this conflict?

I can't reference Xamarin.AndroidX.Lifecycle.LiveData 2.3.1 because then it fails since Xamarin Forms needs < 2.3.0.

````
Version conflict detected for Xamarin.AndroidX.Lifecycle.LiveData. Install/reference Xamarin.AndroidX.Lifecycle.LiveData 2.3.1 directly to project MyApp.Mobile.Droid to resolve this issue. 
 MyApp.Mobile.Droid -> Shiny 2.2.0.2829 -> Shiny.Core 2.2.0.2829 -> Xamarin.AndroidX.Work.Runtime 2.5.0.2 -> Xamarin.AndroidX.Lifecycle.LiveData (>= 2.3.1) 
 MyApp.Mobile.Droid -> Xamarin.Forms 5.0.0.2083 -> Xamarin.AndroidX.Lifecycle.LiveData (>= 2.2.0.4 && < 2.3.0).
````

**Comment by aritchie** [comment, marked as answer]
You can override this by editing the android project and adding the version in question:  <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.3.1" />

You should file your questions under  get tagged that you are a sponsor, so I don't miss them

**Reply by jmichas** [reply]
Ok, cool. I wasn't sure if forcing the update would cause any issues.

**Reply by aritchie** [reply]
It may with XF if this forces other AndroidX UI lib updates.  I can't do anything about that though.  You would have to talk to XF team about this.
